prusti-tests/tests/parse/ui/composite.rs

```diff
@@ -53,4 +53,25 @@ fn test15() {}
 #[requires(forall(|b: i32| b == 10) ==> true ==> forall(|a: u32, b: u32| a == 5))]
 fn test16() {}
 
+#[requires(true && exists(|a: i32| a == 5))]
+fn test17() {}
+
+#[requires(true && exists(|a: i32| a == 5))]
+fn test18() {}
```

Comment:
This one is the same as the previous one?

Reply:
Yes.